lib/windows/loader/components/AppListView.jsx

```diff
@@ -34,12 +34,51 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { Iterable } from 'immutable';
+import AppListItem from './AppListItem';
 
-const AppListView = () => (
-    <div>
-        App list
-    </div>
-);
+function getSortedApps(apps) {
+    return apps.sort((a, b) => {
```

Comment:
I know apps shouldn't have same name, but what happens then? Just wondered about this since this compare function doesn't support returning 0.